content/Software and Downloads/Installation/Add-a-library-to-Arduino-IDE.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Add a library to Arduino IDE"
+---
+
+Libraries are a collection of code that makes it easy to connect sensors, RTCs, Wi-Fi modules, RGB matrices and other components to your board.
+
+---
+
+## Add a library in Library manager
+
+1. In the menu bar, go to `Tools > Manage Libraries...` (In IDE 2, you can also click on the Library Manager icon on the left).
+
+2. Search for the library you want to install.
+
+3. Find the library in the search results.
+
+4. Click on the INSTALL button.
+
+   ![IDE 1 Library Manager with search box and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE1.png)
+
+   ![IDE 2 Library Manager Icon, search box, and Install button highlighted](img/LibraryManager_InstallingLibrary_IDE2.png)
+
+5. Wait for the installation to complete.
+
+6. After the installation is complete, the library will be labelled as INSTALLED, and it is ready to be [included in a sketch](https://www.arduino.cc/reference/en/language/structure/further-syntax/include/).
```

Comment:
This is only one use case. The user's goal can also be to

* satisfy dependency for a sketch the user wants to run (perhaps they're trying to fix a `No such file or directory` error)
* run a library example

I would consider removing this step from from each installation method (the reference is already linked under "further reading").

Reply:
[e03644e](https://github.com/arduino/help-center-content/pull/82/commits/e03644eccf76981b46cc81aac7206f8357cbad20)